x11-libs/cmrt/metadata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>media-video@gentoo.org</email>
+  </maintainer>
+  <maintainer type="person" proxied="yes">
+    <email>kaichun.ning@gmail.com</email>
```

Comment:
Make a bugzilla account with this e-mail, or update the metadata.xml file to have the same e-mail as you use in our bugzilla. Otherwise bugs can't be assigned to you.